tools/CMakeLists.txt

```diff
@@ -1,9 +1,10 @@
 if(NOT TARGET CLI11::CLI11)
+  # https://github.com/CLIUtils/CLI11/releases
   cpmaddpackage(
     NAME
     cli11
     VERSION
-    2.3.2
+    2.4.1
```

Comment:
With GA we are going to package tools too, so CLI parser also have to be tracked, but we don't have BD component yet.